### DISCUSSION TITLE
publishing upm packages has stopped working

### DISCUSSION THREAD

**Discussion by JamesWright1009** [community/community#28347] [Actions]
Hello,

I am working in a company that is using Github Packages as a repository for its Unity UPM packages. As UPM does not actually support @scope notation there is a fairly established workaround that involves leaving the @scope out of the package name and putting it in the publishconfig.registry this worked fine until early on this week when we suddenly started getting this error message:
`
400 Bad Request - PUT https://npm.pkg.github.com/@MyOrg/com.myorg.mypackage - name in package.json "com.myorg.mypackage" does not match publish URL "@MyOrg/com.myorg.mypackage"
`
has something changed recently and is there any way we can work around this. UPM does not support packages with an @ in the name so we cant just change the name in the package.json

Thanks,

James

**Comment by tinaheidinger** [comment]
Hi James, we fixed the error on our side. Can you please confirm that configuring UPM packages works as expected again?

**Comment by mads-modl-ai** [comment]
Hi @tinaheidinger,

We had the same issue as the OP.

The publishing seems to be working again, however the published package "Assets" are no longer available with downloads. This is true also for packages created before the UI got updated 8 days ago! We are no longer able to download our tarballs (.tgz) from specific packaged versions.

[Edit] added clarifying image:
![image](https://user-images.githubusercontent.com/97730170/184327908-b6f45d75-67a3-4bca-81dd-bd62df93e24b.png)

**Comment by tinaheidinger** [comment]
Hi @mads-modl-ai, the download feature was removed due to unwanted side effects. We'll think of other ways how a package's content can be explored without installing it. If you have any thoughts or like to share your use case, please feel free to open a new discussion issue. I'm happy to hear your feedback.

**Reply by mads-modl-ai** [reply]
Hi @tinaheidinger, that's a bit unfortunate, we're looking forward to other ways of downloading the npm package. Meanwhile, we did noticed that the generated npm links on the page incorrectly are all lower case, which means that for instance "PullRequest" version links are incorrectly written as "pullrequest":

![image](https://user-images.githubusercontent.com/97730170/188862552-66b7d279-465e-47eb-b6d8-933f76af5d3f.png)

**Reply by tinaheidinger** [reply]
thanks @mads-modl-ai! we just fixed this. can you please confirm that the install commands are correct now for your npm packages?

**Reply by mads-modl-ai** [reply]
Yeah, this seems to be updated now :)

**Comment by alexandercbooth-test** [comment]
h
